.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1 +1,5 @@
 blank_issues_enabled: false
+contact_links:
+  - name: Discord Server
+    url: https:/discord.gg/google
```

Comment:
Do you really want to link to the r/Google discord for help with this?

Reply:
For now? Why not. Later we may change it but atm the bot is literally just r/google.

Reply:
We can add a different link as a method of contact, such as an email or a discord.bio site. This bot isn't strictly affiliated with r/Google, let's change it

Reply:
> We can add a different link as a method of contact, such as an email or a discord.bio site. This bot isn't strictly affiliated with r/Google, let's change it

I agree @benfoster04, The bot isnt directly associated with r/Google (and for sure the raw code) and there is no reason any of the r/Google staff, or server members should be associated with it. The only bot we should assist with is the instance in the server. The only contact method for help with this code would be the issues pages, maybe we will work on something else later down the line, but that is the current case. I mirroed this fact in the security page, and in SUPPORT.md